### DISCUSSION TITLE
Where to locate

### DISCUSSION THREAD

**Discussion by Ocim069** [coder/code-server#3235] [Q&A] [answered]
I tried to run vscode on my android phone but I can't locate configs.yaml where the password is stored. 

Can anyone help me locate configs.yaml so I can see my default password and login to codeserver.

Thanks! :)

**Comment by nooobcoder** [comment, marked as answer]
First, browse to this directory
`cd ~/.config/code-server/`
and do list all files in the directory
`ls -a`
If config.yaml does not exist create it with some util tools like
`vim` `nano` `echo`
Also, you can create the file first by using, `touch config.yaml` @Ocim069 

Please post any updates you have, thank you.
